components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -293,8 +307,8 @@ const EditVisibleAssetsModal = (props: Props) => {
                   </>
                 </WatchlistScrollContainer>
                 <NavButton
-                  onSubmit={toggleShowAddCustomToken}
-                  text={getLocale('braveWalletWatchlistAddCustomAsset')}
+                  onSubmit={onClose}
```

Comment:
Is the Done button doing nothing? I think the expectation from such UX is that if you don't click on Done (and just close the modal), then no changes made will be applied, which doesn't seem to be the case.

Is it possible to make Done behave in a way that applies all the changes in one go? If it's too much work, we can keep it as-is, but in that case we might as well not display the Done button at all.

Reply:
Per slack discussion, currently the api is not set up to handle multiple adding/removing of tokens at once, it has to be handled 1 at a time.

Reply:
Pls post a follow up for the behavior that @onyb suggested.  It's not MVP blocking but what he's suggesting is best.

Reply:
Posted an issue here: https://github.com/brave/brave-browser/issues/18532